types/array/array.ts

```diff
@@ -1,62 +1,72 @@
-import { SizedType, ViewableType } from "../types.ts";
+import { SizedType, TypeOptions, ViewableType } from "../types.ts";
 
 export class ArrayType<T> implements SizedType<T[]>, ViewableType<T[]> {
   byteLength: number;
-  byteStride: number;
+  bytesPerElement: number;
```

Comment:
```suggestion
  byteStride: number;
```

I quite like calling it `byteStride` as it is what the step size in an array is commonly called (https://en.wikipedia.org/wiki/Stride_of_an_array). The stride could very possibly be larger than the element size due to alignment to make it performant.